### ISSUE TITLE
Loader crashes when no options are supplied

### ISSUE THREAD

**Issue by billyjanitsch** [babel/babel-loader#412]
**I'm submitting a bug report**

**Webpack Version:**
2.2.1

**Babel Core Version**:
6.23.1

**Babel Loader Version**:
7.0.0-alpha.2

**Please tell us about your environment:**
OSX 10.12.3

**Current behavior:**
babel-loader crashes when no options are supplied:
```
Module build failed: TypeError: Cannot read property 'babelrc' of null
```

**Expected/desired behavior:**
Supplying no options should be supported.

* **If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem along with a gist/jsbin of your webpack configuration.**
```js
{
  test: /\.js$/,
  use: 'babel-loader',
  // don't pass any options to the loader
}
```

The issue is that [loader-utils' `getOptions`](https://github.com/babel/babel-loader/blob/7.0/src/index.js#L96) returns `null` if no options are supplied to the loader, so you can't [access properties](https://github.com/babel/babel-loader/blob/7.0/src/index.js#L105) assuming that it will be an object. Appending `|| {}` to L96 should fix it, but I haven't had a chance to verify.

**Comment by pugnascotia** [comment]
I'm seeing this too. @billyjanitsch's analysis correlates with what I found.

**Comment by danez** [comment]
Ok I fixed it in 7.0.0-alpha.3

**Comment by billyjanitsch** [comment]
Thanks for the quick fix!
